### PR TITLE
feat: add api routes

### PR DESCRIPTION
**Pull request by junaidrahim**
Add api routes `/` `/:path` and `/hook`. Also made modifications to the cli commands

## Pre-launch Checklist

- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [x] Code builds locally.
- [x] Code is Formatted gofmt.

### CODE REVIEW

**Review by rohankmr414 — APPROVED** [review]
LGTM

File: api/routes.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package api
```
Comment (amanv8060) [inline api/routes.go:1]: please add license headers.

File: utils/utils.go [hunk 2]
```diff
@@ -1 +1 @@
-package utils
\ No newline at end of file
+package utils
```
Comment (amanv8060) [inline utils/utils.go:1]: here as well.

File: api/api.go [hunk 3]
```diff
@@ -1,3 +1,8 @@
+/*
+ * Copyright 2022 The ProDocs Authors. All rights reserved. Use of this source code
```
Comment (amanv8060) [inline api/api.go:2]: wrong formatting.

**Review by amanv8060 — CHANGES_REQUESTED** [review]
nit: improve License headers in general.

**Pull request merged**